src/editor/screens/AddActivityCard.js

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import {
+  SafeAreaView,
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  useState
+} from 'react-native';
+import TagFilter from '../components/TagFilter';
+import BackArrow from '../../../assets/backArrow.svg';
+import { Searchbar } from 'react-native-paper';
+import { mdiMagnify } from '@mdi/js';
+import { isSearchBarAvailableForCurrentPlatform } from 'react-native-screens';
+
+const TAGS = [
+  'Warm Up',
+  'No Equipment',
+  'Beginner',
+  'Rhythm',
+  'Note Reading',
+  'Group Activity',
+  'Quick',
+  'Scale'
+];
+
+const AddActivityCard = function () {
+  const [searchQuery, setSearchQuery] = React.useState('');
+
+  const onChangeSearch = query => setSearchQuery(query);
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <View style={{ paddingHorizontal: '3%' }}>
+        <TouchableOpacity style={styles.backButton}>
+          <BackArrow height={15} width={15} />
+          <Text style={styles.backText}> Back </Text>
+        </TouchableOpacity>
+        <Text style={styles.header}> Main Lesson Activity Card </Text>
+        <Text style={styles.tags}> Tags: </Text>
+        <View style={styles.tagContainer}>
+          {TAGS.map((tag, index) => (
+            <TagFilter key={index} tagContent={tag} />
+          ))}
+        </View>
+        <Searchbar
+          placeholder="Search by title or keyword"
+          onChangeText={onChangeSearch}
+          value={searchQuery}
+          icon={require('../../../assets/search.png')}
+          style={{ backgroundColor: 'white', marginTop: '3%' }}
```

Comment:
consider moving to styles -- @yuemily24 should we avoid using inline styles or is it okay for small styling in individual components

Reply:
I'm not sure how accurate this is but I think emily mentioned to use SafeAreaView instead of View??

Reply:
I can move to styles, no problem. As for @Lemiex the AddActivityCard screen is wrapped in SafeAreaView so it should be fine to have Views nested inside it.